flashlight/fl/examples/RnnClassification.cpp

```diff
@@ -103,10 +103,10 @@ class ClassificationDataset : public Dataset {
         "Irish"};
     for (auto& l : lang) {
       read(datasetPath, l);
-}
+    }
```

Comment:
would you mind adding `explicit` to the `ClassificationDataset` constructor too? :D

Reply:
Sure!